app/Http/Controllers/Admin/SettingsController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\Setting;
+use Illuminate\Http\Request;
+
+class SettingsController extends Controller
+{
+    public function index()
+    {
+        $settings = Setting::all();
+        return view("admin.settings.settings", compact("settings"));
+    }
+
+    public function edit(Request $request)
+    {
+        $setting = Setting::where("key", $request->key)->update(["value" => $request->value]);
+        if ($setting) {
+            return "Başarılı";
+        }
+        return "hatalı";
+    }
+
+    public function store(Request $request)
+    {
+        $setting = new Setting();
+
+        $setting->key = $request->key;
+        $setting->value = $request->value;
+
+        if ($setting->save()) {
+            return ["status" => "success", "message" => "Ayar başarıyla kaydedildi", "title" => "Başarılı"];
+        } else {
```

Comment:
gereksiz if else kullanımlarından kaçınman gerekiyor. early return kavramını öğrenirsen işin çok daha kolaylaşır ve kodların daha okunabilir hale gelir. https://freek.dev/1593-writing-readable-php-decrease-indentation-by-returning-early

https://dev.to/jpswade/return-early-12o5

https://pear.php.net/manual/en/standards.bestpractices.php

https://szymonkrajewski.pl/why-should-you-return-early/